api/v2/dogu_types.go

```diff
@@ -216,16 +221,17 @@ func (ds *DoguStatus) ResetRequeueTime() {
 //
 //goland:noinspection GoUnusedConst
 const (
-	DoguStatusNotInstalled       = ""
-	DoguStatusInstalling         = "installing"
-	DoguStatusUpgrading          = "upgrading"
-	DoguStatusDeleting           = "deleting"
-	DoguStatusInstalled          = "installed"
-	DoguStatusPVCResizing        = "resizing PVC"
-	DoguStatusStarting           = "starting"
-	DoguStatusStopping           = "stopping"
-	DoguStatusChangingExportMode = "changing export-mode"
-	DoguStatusChangingDataMounts = "change data mounts"
+	DoguStatusNotInstalled                        = ""
+	DoguStatusInstalling                          = "installing"
+	DoguStatusUpgrading                           = "upgrading"
+	DoguStatusDeleting                            = "deleting"
+	DoguStatusInstalled                           = "installed"
+	DoguStatusPVCResizing                         = "resizing PVC"
+	DoguStatusStarting                            = "starting"
+	DoguStatusStopping                            = "stopping"
+	DoguStatusChangingExportMode                  = "changing export-mode"
+	DoguStatusChangingDataMounts                  = "change data mounts"
+	DoguStatusConditionMeetsMinimumDataVolumeSize = "MeetsMinimumDataVolumeSize"
```

Comment:
Please use a shorter names and use lower case for the key in the Dogu CR. We can drop `DoguStatus` as a prefix here as conditions will become its own thing.
```suggestion
	ConditionMeetsMinVolumeSize = "meetsMinVolumeSize"
```